humanitec-resource-defs/iam-role-binding/gcs/main.tf

```diff
@@ -0,0 +1,23 @@
+resource "humanitec_resource_definition" "main" {
+  driver_type = "humanitec/template"
+  id          = "${var.prefix}iam-role-biding-gcs-${var.name}"
+  name        = "${var.prefix}iam-role-biding-gcs-${var.name}"
+
+  # TODO Switch the GCP type once available
+  type = "aws-policy"
```

Comment:
This still needs to be changed once there is a proper type, but merging so we can iterate on main.